src/Patch/Node.js

```diff
@@ -154,7 +154,8 @@ export default function(internals) {
         return nativeResult;
       }
 
-      const nodeToInsertWasConnected = Utilities.isConnected(nodeToInsert);
+      const nodeToInsertWasConnected = nodeToInsert instanceof Element &&
```

Comment:
`nodeToInsertWasConnectedElement`, for consistency with the rest of the renaming

Reply:
Done.